src/platform_impl/linux/window.rs

```diff
@@ -402,6 +403,16 @@ impl Window {
     }
   }
 
+  /// Set menu
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Windows/Linux:** Unsupported (noop).
+  ///
+  pub fn set_menu(&self, _menu: Option<Vec<Menu>>) {
+    debug!("`Window::set_menu` is ignored on linux")
```

Comment:
I can help implementing this.

Reply:
![image](https://user-images.githubusercontent.com/8409985/117103019-1ef0a680-adac-11eb-91f3-b853d46089cf.png)
Initial implementation looks promising. Do you want me to push it or wait until next PR?

Reply:
Sure!

Reply:
great! I'll push some implementation first.